pennylane/__init__.py

```diff
@@ -63,6 +63,8 @@
     mutual_info,
     classical_shadow,
     shadow_expval,
+    Shots,
+    ShotCopies,
```

Comment:
Do we need to import these top-level? Most users won't be specifying `Shots` as a shot class, but as one of the possible inputs to `Shots`.